internal/host/hostcommands/install_cmd.go

```diff
@@ -305,15 +308,22 @@ func constructHostInstallerArgs(cluster *common.Cluster, host *models.Host, inve
 		}
 	}
 
+	var installationDisk *models.Disk
 	for _, disk := range inventory.Disks {
 		if disk.ID == host.InstallationDiskID {
-			if disk.DriveType == models.DriveTypeMultipath {
-				installerArgs = append(installerArgs, "--append-karg", "root=/dev/disk/by-label/dm-mpath-root", "--append-karg", "rw", "--append-karg", "rd.multipath=default")
-			} else if disk.DriveType == models.DriveTypeISCSI {
-				// Currently only allowed on the OCI platform
-				installerArgs = append(installerArgs, "--append-karg", "rd.iscsi.firmware=1")
-			}
+			installationDisk = disk
+			break
+		}
+	}
```

Comment:
1.  what happens if no installation disk has been found? 
2. maybe we should create a common function to get installation disk and error like `common.GetInstallatioDisk(disks []models.Disk) models.Disk error`

Reply:
old behavior was just ignoring the missing installation disk. But a warning would be nice